base_layer/wallet_ffi/python_build.sh

```
@@ -0,0 +1,89 @@
+    echo "========================================="
+    echo "Building for $network network..."
+    echo "========================================="
+    
+    # Set network environment variable
+    export TARI_TARGET_NETWORK=$network
+    
+    # Build the wheel
+    maturin build --features python-bindings --release
+    
+    # Find the generated wheel (maturin creates it in workspace target/wheels)
+    local wheel_file=$(find ../../target/wheels -name "tari_wallet-*.whl" -type f 2>/dev/null | head -n 1)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)